packages/components/src/ActionBar/__snapshots__/ActionBar.snapshot.test.js.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ActionBar should render a btn-group 1`] = `
-<ActionBar
+<I18n
```

Comment:
This makes the snapshot ko.
Now instead of having an `ActionBar`, we have an `I18n`. Perhaps you should import the component with the translate HOC for the tests.

(same for all components)